ci/drop-canton.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+set -xeuo pipefail
+
+date=$(date -u "+%Y-%m-%d %H:%M")
+canton_dir=canton
+canton_repo="git@github.com:DACH-NY/canton.git"
+
+DEBUG=1
+
+err() {
+  (>&2 echo "ERROR: $1")
+}
+
+debug() {
+  [ -z "$DEBUG" ] || (>&2 echo "DEBUG: $1")
+}
+
+run() {
+  local message="$1"
+  shift
+  local -ra cmd=( "$@" )
+  local output
+
+  debug "$message [${cmd[*]}]"
+
+  if ! output=$("$@" 2>&1); then
+    err "$message failed: $output"
+    exit 1
+  fi
+}
+
+function refresh() {
```

Comment:
I tend to [prefer subshell functions](https://cuddly-octo-palm-tree.com/posts/2021-10-31-better-bash-functions/).